src/modules/functions.js

```diff
@@ -40,9 +40,17 @@ export const refresher = async (where) => {
   );
   const messageJson = await data.text();
   const message = await JSON.parse(messageJson);
+  let i = 0;
+  let colorClass = '';
   message.result.forEach((element) => {
+    i += 1;
+    if (i % 2 === 0) {
+      colorClass = 'color';
+    } else {
+      colorClass = '';
```

Comment:
- [ ] [Optional] Instead of using the `color` class, you can use the `nth-child(even)` CSS selector on the `li`s